test/integration/helpers/transferDuplicateCheck.js

```diff
@@ -78,16 +78,3 @@ exports.prepareData = async () => {
     throw ErrorHandler.Factory.reformatFSPIOPError(err)
   }
 }
-
-// exports.deletePreparedData = async (transferId, payerName, payeeName) => {
```

Comment:
I am not sure this should be removed. I suspect it may be useful. I am also not sure why its commented out to be honest.

Reply:
Re-added it, but still commented out.